org.metaborg.spoofax.shell.console/src/test/java/org/metaborg/spoofax/shell/client/console/impl/ConsoleReplTest.java

```diff
@@ -85,10 +84,10 @@ private void setUp(String inputString) throws UnsupportedEncodingException {
 
     private void setUpCtrlD() throws IOException {
         setUp(String.valueOf(C_D));
-        invokerMock = mock(ICommandInvoker.class, RETURNS_MOCKS);
+        invokerSpy = mock(ICommandInvoker.class, RETURNS_MOCKS);
```

Comment:
Ah, I see now why I changed this. In this test, it is indeed a mock. In `setUpExit()`, it is a spy. Anyway, everything is now just named `mock` again.